packages/analytics/src/functions.ts

```diff
@@ -142,3 +143,27 @@ export async function setAnalyticsCollectionEnabled(
   const measurementId = await initializationPromise;
   window[`ga-disable-${measurementId}`] = !enabled;
 }
+
+/**
+ * Consent parameters to default to during 'gtag' initialization.
+ */
+export let defaultConsentSettingsForInit: ConsentSettings | undefined;
+
+/**
+ * Sets the variable {@link defaultConsentSettingsForInit} for use in the initialization of
+ * analytics.
+ *
+ * @param consentSettings Maps the applicable end user consent state for gtag.js.
+ */
+export function _setConsentDefaultForInit(
```

Comment:
I'm assuming you'll want me to change this to just overwriting the existing consent details like in https://github.com/firebase/firebase-js-sdk/blob/16b481619e851c2f1577fbc0b3e7d280f3b0087d/packages/analytics/src/functions.ts#L158

Reply:
Yes, thanks.